portal-web/test/functional/com/liferay/portalweb/tests/Homepage.testcase

```diff
@@ -0,0 +1,37 @@
+definition {
+
+	property testray.main.component.name = "Login";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+		}
+	}
+
+	@description = "LRQA-82341 Post Login Admin can view Welcome to Liferay text"
+	test VerfiyUserName {
```

Comment:
@Piyush-Gupta10 typo in test name.  And using the SACRED model of testing, we include also the expected outcome in the test name.  So something like AdminCanViewHomepageWelcomeToLiferay would be a better name.  The test name and description can be closely related.